lib/flatpak-installation.c

```diff
@@ -973,26 +988,31 @@ list_remotes_for_configured_remote (FlatpakInstallation  *self,
 }
 
 /**
- * flatpak_installation_list_remotes:
+ * flatpak_installation_list_remotes_by_type:
  * @self: a #FlatpakInstallation
+ * @types: an array of #FlatpakRemoteType
```

Comment:
Add an annotation to say the array length is given by `num_types`: `(array length=num_types)`.